actors/builtin/miner/miner_state.go

```diff
@@ -191,12 +172,37 @@ func ConstructState(emptyArrayCid, emptyMapCid, emptyDeadlinesCid cid.Cid, owner
 	}, nil
 }
 
-func (st *State) GetWorker() addr.Address {
-	return st.Info.Worker
+func (st *State) GetInfo(store adt.Store) (*MinerInfo, error) {
+	var info MinerInfo
+	if err := store.Get(store.Context(), st.Info, &info); err != nil {
+		return nil, errors.Wrapf(err, "failed to get miner info")
```

Comment:
Don't extend the use of `pkg/errors`, use `%w` either with `fmt.Errorf` or `xerrors`.